README.md

```diff
@@ -1,8 +1,33 @@
-# BA
-My Bachelorthesis Repository
+# A cuboid muscle model
 
-**Content:**
+## Setup
+- A dummy cuboid muscle geometry. 
+- Solver comprises hyperelasticity solver for the prestretch and both mechanics solver + fastmonodomain solver for the contraction. 
```

Comment:
no longer true, it is a FastMonodomainSolver + MuscleContractionSolver. See the [readme](https://github.com/opendihu/opendihu/blob/95fbdfe31be284697846d218570726ac544b27ec/examples/electrophysiology/neuromuscular/cuboid_muscle_with_prestretch/README.md) I wrote for the opendihu repo

Reply:
Yes, I have added this now.